### ISSUE TITLE
Create "report page" of each Civic Association

### ISSUE THREAD

**Issue by icanhazcodeplz** [DataKind-DC/EcoAction#21]
There is a good chance interactive plots online will be too slow on EcoAction computers. We might want to make a "report", in which there could be one page per civic association (CA) that provides stats. Some initial ideas for each page:

* Show one or two maps that are zoomed into that specific CA, perhaps:
    * Open and plantable area
    * Block-groups (BG) within the CA
    * Locations of trees EcoAction has previous planted
* Statistics: one table for the civic association, perhaps separate tables for each BG within the CA
    * Demographics: population density, %nonwhite, %in_poverty, others?
    * Land area: canopy per person, %canopy, %open_plantable
    * Previously planted trees: number of trees (broken down by year? broken down by tree-type?)

**Comment by icanhazcodeplz** [comment]
I'll make some mock-ups of these ideas today. Might even use fake data just to get something to demo to EcoAction.

**Comment by icanhazcodeplz** [comment]
Mmkay, I sketched up some ideas in a google slides. Look for `Arlington Trees Report: Civic Associations` in the **EcoAction Arlington/Deliverables** folder in google drive. I just did it for a single Civ Assoc, and it was all done manually. I want to get some feedback from EcoAction before spending time automating it. My code is not clean or presentable yet, but I did save my script `scripts/civic_association_report_page.R` in the branch `civ_assoc_report_page`.

I also added the google sheet `Civic Association Stats (estimated)` in the same folder
